### PR TITLE
Review branch merge

### PR DESCRIPTION
**Pull request by xeoneux**

### CODE REVIEW

File: assets/icons/location.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default () => (
+  <svg width="24px" height="24px" baseProfile="tiny" viewBox="0 0 24 24">
+    <g>
+      <path d="M10.368,19.102c0.349,1.049,1.011,1.086,1.478,0.086l5.309-11.375c0.467-1.002,0.034-1.434-0.967-0.967L4.812,12.154   c-1.001,0.467-0.963,1.129,0.085,1.479L9,15L10.368,19.102z" />
+    </g>
+  </svg>
+);
```
Comment (sunil-jhamnani) [inline assets/icons/location.jsx]: Why create react component for returning SVG's?

File: components/info.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import UserCard from "react-ui-cards/src/UserCard";
+
+const truncate = string => {
+  const length = 75;
```
Comment (sunil-jhamnani) [inline components/info.jsx]: Should go in constants so that its configurable

File: components/layout.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import styled from "@emotion/styled";
+
+import Map from "./map";
+import Logo from "./logo";
+import Places from "./places";
+import Location from "./location";
+import Global from "../styles/global";
+
+import "sanitize.css";
+
```
Comment (sunil-jhamnani) [inline components/layout.jsx]: Can we add sanitize.css in Global?

File: assets/icons/location.jsx (hunk 1 above)
Comment (xeoneux) [reply assets/icons/location.jsx]: It's easier to pass in variables to change properties and add react's event listeners directly to the svg component.

File: components/info.jsx (hunk 2 above)
Comment (xeoneux) [reply components/info.jsx]: Changes made 👍

File: components/layout.jsx (hunk 3 above)
Comment (xeoneux) [reply components/layout.jsx]: Changes made 👍

File: package.json [hunk 4]
```diff
@@ -1,8 +1,43 @@
...
+    "next-transpile-modules": "^2.2.0",
+    "node-sass": "^4.11.0",
+    "prop-types": "^15.7.2",
     "react": "^16.8.4",
-    "react-dom": "^16.8.4"
+    "react-dom": "^16.8.4",
+    "react-geolocated": "^2.4.1",
+    "react-map-gl": "^4.0.14",
+    "react-select": "^2.4.1",
+    "react-ui-cards": "nukeop/react-ui-cards",
```
Comment (sunil-jhamnani) [inline package.json]: Don't use modules which might cause console errors
react-ui-cards is one of them

File: components/map.jsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+              {...marker}
+              onClick={() => this.setState({ popupInfo: marker })}
+            />
+          </Marker>
+        ))}
+        {this.renderPopup()}
+      </ReactMapGL>
+    );
+  }
+}
```
Comment (sunil-jhamnani) [inline components/map.jsx]: No props validation

File: components/info.jsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+export default class Info extends React.PureComponent {
+  static propTypes = {
+    icon: PropTypes.string,
+    photo: PropTypes.string,
+    name: PropTypes.string.isRequired,
+    rating: PropTypes.number.isRequired,
+    vicinity: PropTypes.string.isRequired,
+    latitude: PropTypes.string.isRequired,
+    longitude: PropTypes.string.isRequired,
+    totalRatings: PropTypes.number.isRequired
```
Comment (sunil-jhamnani) [inline components/info.jsx]: Invalid prop `latitude` of type `number` supplied to `Info`, expected `string`.
Console error

File: components/info.jsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+import PropTypes from "prop-types";
+import UserCard from "react-ui-cards/src/UserCard";
+
+import DESC_LENGTH from "../helpers/constants";
+
+const truncate = string => {
+  const length = DESC_LENGTH;
+  return string.length > length ? `${string.substring(0, length)}...` : string;
+};
+
```
Comment (sunil-jhamnani) [inline components/info.jsx]: Can be a utility/common function

File: components/map.jsx (hunk 5 above)
Comment (xeoneux) [reply components/map.jsx]: Fixed 👍

File: components/info.jsx (hunk 6 above)
Comment (xeoneux) [reply components/info.jsx]: Fixed 👍

File: components/info.jsx (hunk 7 above)
Comment (xeoneux) [reply components/info.jsx]: Fixed 👍

**Pull request merged**